Whiskey/Functions/Test-WhiskeyPowerShellModule.ps1

```diff
@@ -0,0 +1,52 @@
+function Test-WhiskeyPowershellModule
+{
+    <#
+    .SYNOPSIS
+    Checks if a module is installed globally.
+     
+    .DESCRIPTION
+    The `Test-WhiskeyPowerShellModule` function tests if a module is installed globally. If the module is installed globally, it returns true. Otherwise it returns false.

```

Comment:
Update the docs to describe _how_ the module is searched for globally. E.g. `Get-Module -ListAvailable` is used to search which looks for installed modules in the list of paths found in the [PSModulePath](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_psmodulepath?view=powershell-5.1) environment variable.